tests/test_main.py

```diff
@@ -11,7 +11,7 @@
     direct_write=True,
     direct_write_mode='serial',
     #printer_port="/dev/tty.usbmodem1411",
-    printer_port="COM8",
+    printer_port="COM14",
```

Comment:
We discussed making the COM port a command line argument. I can do this tomorrow.